src/systemtest/java/reposense/LocalRepoSystemTest.java

```diff
@@ -63,16 +64,29 @@ public static void deleteClonedLocalRepos() throws Exception {
 
     @Test
     public void testSameFinalDirectory() {
-        String cliInput = String.format("-r %s %s -s d1 -u %s -o %s -t %s",
-                LOCAL_DIRECTORY_ONE, LOCAL_DIRECTORY_TWO, LAST_COMMIT_DATE, OUTPUT_DIRECTORY, TIME_ZONE);
+        InputBuilder inputBuilder = new InputBuilder()
+                .addRepos(LOCAL_DIRECTORY_ONE, LOCAL_DIRECTORY_TWO)
+                .addSinceDate(EARLIEST_COMMIT_DATE)
+                .addUntilDate(LAST_COMMIT_DATE)
+                .addOutput(Paths.get(OUTPUT_DIRECTORY))
+                .addTimezone(TIME_ZONE);
+
+        String cliInput = inputBuilder.build();
         runTest(cliInput, "LocalRepoSystemTest/testSameFinalDirectory");
```

Comment:
A bit weird to have some preprocessing here and then another `String[] args = commandArgs.split(" ");` before `RepoSense.main(args);`.

I suggest an alternative way which is to pass the `inputBuilder` to `runTest()`. You can refer to [this](https://github.com/reposense/RepoSense/pull/1816/files#diff-59400ca2297b765e07badbf3424d7ac8fdb8237d902ea99ad60a581447f9b05cR175) to see how it is done.